ui/components/MesheryPerformanceComponent.js

```diff
@@ -613,6 +558,77 @@ class MesheryPerformanceComponent extends React.Component {
                   </RadioGroup>
                 </FormControl>
               </Grid>
+              <Grid item xs={12} sm={12} gutterBottom>
+                <ExpansionPanel className={classes.expansionPanel}>
+                  <ExpansionPanelSummary expanded={true}>
+                    <Typography align="center" color="textSecondary" varient="h6">More Options</Typography>
```

Comment:
Also, can we remove the border structure from the More Options & add an arrow-down/up icon on the basis of the state of the expansion panel?
<img width="935" alt="Screenshot 2020-06-19 at 7 59 17 PM" src="https://user-images.githubusercontent.com/44091822/85143646-a05c4a80-b267-11ea-8c62-5acb144e29af.png">

Somewhat like this with an arrow icon adjacent to it.
<img width="939" alt="Screenshot 2020-06-19 at 8 02 40 PM" src="https://user-images.githubusercontent.com/44091822/85143805-de596e80-b267-11ea-9dd4-acbd7f628ae5.png">